# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/wmx/compilefix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN wmx-6/Menu.C wmx-6-avm/Menu.C
--- wmx-6/Menu.C	2000-05-24 17:50:27.000000000 +0200
+++ wmx-6-avm/Menu.C	2003-06-13 23:17:17.000000000 +0200
@@ -621,7 +621,7 @@
 
 
 CommandMenu::CommandMenu(WindowManager *manager, XEvent *e,
-			 char* otherdir = NULL)
+			 char* otherdir)
     : Menu(manager, e)
 {
     const char *home = getenv("HOME");
